triangle_facts/triangle_facts.rb

```diff
@@ -39,20 +24,45 @@ def calculate_angles(a,b,c)
 		angleB = radians_to_degrees(Math.acos((a**2 + c**2 - b**2) / (2.0 * a * c)))
 		angleC = radians_to_degrees(Math.acos((a**2 + b**2 - c**2) / (2.0 * a * b)))
 
-		return [angleA, angleB, angleC]
+		[angleA, angleB, angleC]
 	end
 
 	def radians_to_degrees(rads)
-		return (rads * 180 / Math::PI).round
+		(rads * 180 / Math::PI).round
+	end
+
+	def type_facts
+		case
+		when equilateral
+			puts "This triangle is equilateral!"
+		when isosceles
+			puts "This triangle is isosceles! Also, that word is hard to type."
+		when scalene
+			puts "This triangle is scalene and mathematically boring."
+		end
+	end
+
+	def angle_facts
+		angles = calculate_angles(side1, side2, side3)
+		puts "The angles of this triangle are " + angles.join(',')
+		if angles.include?(90)
+			puts "This triangle is also a right triangle!"
+		end
+	end
+
+	def recite_facts
+		print type_facts
+		puts angle_facts
```

Comment:
`type_facts` and `angle_facts` are already printing stuff so these prints are confusing. Two solutions:
1. Remove these print statements and rename the methods to indicate they are printing
2. Change the underlying methods to return the text instead of printing